dotCMS/src/main/java/com/dotmarketing/fixtask/tasks/FixTask00095DeleteOrphanRelationships.java

```diff
@@ -0,0 +1,173 @@
+package com.dotmarketing.fixtask.tasks;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.sql.SQLException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import com.dotcms.repackage.com.thoughtworks.xstream.XStream;
+import com.dotcms.repackage.com.thoughtworks.xstream.io.xml.DomDriver;
+import com.dotmarketing.beans.FixAudit;
+import com.dotmarketing.beans.Inode;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.fixtask.FixTask;
+import com.dotmarketing.portlets.cmsmaintenance.ajax.FixAssetsProcessStatus;
+import com.dotmarketing.portlets.cmsmaintenance.factories.CMSMaintenanceFactory;
+import com.dotmarketing.util.ConfigUtils;
+import com.dotmarketing.util.Logger;
+
+
+public class FixTask00095DeleteOrphanRelationships implements FixTask{
+
+    private List<Map<String, String>> modifiedData = new ArrayList<Map<String, String>>();
+
+    /** Lookup invalid inodes in Field table referencing the Structure table */
+    private static final String VERIFICATION_QUERY = "SELECT inode FROM relationship WHERE NOT EXISTS " +
+            "(SELECT * FROM structure WHERE relationship.parent_structure_inode = structure.inode) " +
+            " OR NOT EXISTS " +
+            "(SELECT * FROM structure WHERE relationship.child_structure_inode = structure.inode) ";
+
+    @Override
+    public List<Map<String, Object>> executeFix() throws DotDataException,
+            DotRuntimeException {
+        Logger.info(FixTask00095DeleteOrphanRelationships.class,
+                "Beginning DeleteOrphanRelationships");
+        List<Map<String, Object>> returnValue = new ArrayList<Map<String, Object>>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)